udata/api/__init__.py

```diff
@@ -165,12 +168,7 @@ def page_parser(self):
 
 api = UDataApi(
     apiv1,
-    decorators=[csrf.exempt,
-                cors.crossdomain(origin='*',
-                                 credentials=True,
```

Comment:
Credentials are set to true here.
Flask-cors has an option 'supports_credentials' but as the documentation states 'This option cannot be used in conjuction with a ‘*’ origin'.
Do we need to refine the origin parameter in order to use this option?

Reply:
good question, let's see what works and doesn't work on demo as is.

Reply:
(I don't see how we can refine the origin 🤔)